src/services/admin-api.ts

```diff
@@ -61,8 +61,13 @@ export interface ElevatorBooking {
   phoneDay: string;
   phoneNight: string
   deposit: number;
-  moveType: string;
+  moveType: number;
+  moveIn: boolean;
```

Comment:
Similar questions in condo-api PR review - is there a reason for not also including disposal/delivery bools?

Reply:
Because I should delete delivery and disposal. 🤪

It's basically the same question (in, out, both?).